Description: Look for the systemd functions in the new libraries.
Origin: upstream; http://www.stunnel.org/pipermail/stunnel-users/2014-October/004810.html
Last-Update: 2014-10-16

--- a/configure.ac
+++ b/configure.ac
@@ -192,8 +192,8 @@
 # Checks for dynamic loader needed by OpenSSL
 AC_SEARCH_LIBS([dlopen], [dl])
 AC_SEARCH_LIBS([shl_load], [dld])
-AC_SEARCH_LIBS([sd_listen_fds], [systemd-daemon])
-
+# the library name has changed to -lsystemd in systemd 209
+AC_SEARCH_LIBS([sd_listen_fds], [systemd systemd-daemon])
 
 # Add BeOS libraries
 if test "$host_os" = "beos"; then
